src/jvm/test/resources/output/switch_as_first_method_event.txt

```diff
@@ -0,0 +1,34 @@
+= Invalid execution results =
```

Comment:
Please consider embedding these outputs in the test source files, e.g., adding the corresponding string constants at the end of the files. It will make tests easier to maintain.

You may like the power of multiline strings: https://kotlinlang.org/docs/coding-conventions.html#strings

Reply:
Due to the [problem](https://youtrack.jetbrains.com/issue/KT-2425) of escaping dollar sign in kotlin's multiline strings I think it's better to keep this approach to store expected output.